public/app/plugins/panel/gettingstarted/steps.ts

```diff
@@ -0,0 +1,109 @@
+const step2Key = `${keyPrefix}${step2TutorialTitle
+  .replace(' ', '-')
+  .trim()
+  .toLowerCase()}`;
+
+export const getSteps = (): SetupStep[] => [
+  {
+    heading: 'Welcome to Grafana',
+    subheading: 'The steps below will guide you to quickly finish setting up your Grafana installation.',
+    title: 'Basic',
+    info: 'The steps below will guide you to quickly finish setting up your Grafana installation.',
+    done: false,
+    cards: [
+      {
+        type: 'tutorial',
+        heading: 'Data source and dashboards',
+        title: step1TutorialTitle,
+        info:
+          'Set up and understand Grafana if you have no prior experience. This tutorial guides you through the entire process and covers the “Data source” and “Dashboards” steps to the right.',
+        href: 'https://grafana.com/tutorials/grafana-fundamentals?utm_source=grafana_gettingstarted',
+        icon: 'grafana',
+        check: () => Promise.resolve(store.get(step1Key)),
+        key: step1Key,
+        done: false,
+      },
+      {
+        type: 'docs',
+        title: 'Add your first data source',
+        heading: 'data sources',
+        icon: 'database',
+        learnHref:
+          'https://grafana.com/docs/grafana/latest/features/datasources/add-a-data-source?utm_source=grafana_gettingstarted',
+        href: 'datasources/new?utm_source=grafana_gettingstarted',
+        check: () => {
+          return new Promise(resolve => {
+            resolve(
+              getDatasourceSrv()
+                .getMetricSources()
+                .filter(item => {
+                  return item.meta.builtIn !== true;
+                }).length > 0
+            );
+          });
+        },
+        done: false,
+      },
+      {
+        type: 'docs',
+        heading: 'dashboards',
+        title: 'Create your first dashboard',
+        icon: 'apps',
+        href: 'dashboard/new?utm_source=grafana_gettingstarted',
+        learnHref:
+          'https://grafana.com/docs/grafana/latest/guides/getting_started/#create-a-dashboard?utm_source=grafana_gettingstarted',
+        check: async () => {
+          const result = await getBackendSrv().search({ limit: 1 });
+          return result.length > 0;
+        },
+        done: false,
+      },
+    ],
+  },
+  {
+    heading: 'Setup complete!',
+    subheading:
+      'All necessary steps to use Grafana are done. Now tackle advanced steps or make the best use of this home dashboard – it is, after all, a fully customizable dashboard – and remove this panel.',
+    title: 'Advanced',
+    info: ' Manage your users and teams and add plugins. These steps are optional',
+    done: false,
+    cards: [
+      {
+        type: 'tutorial',
+        heading: 'Users',
+        title: 'Create users and teams',
+        info: 'Learn to organize your users in teams and manage resource access and roles.',
+        href: 'https://grafana.com/tutorials/create-users-and-teams?utm_source=grafana_gettingstarted',
```

Comment:
can we add this "utm_source=grafana_gettingstarted" in the render step as well so it does not need to be part of every item